### PR TITLE
fix(simcard): add subscriber_id as legitimate data and keep data

### PR DESCRIPTION
**Pull request by stonebuzz**
Add ```subscriber_id``` as legitimate data (ivnentory format)

Keep ```serial```, and ```subscriber_id``` from inventory send to GLPI

Usefull for https://github.com/glpi-project/glpi/pull/15541

### CODE REVIEW

**Comment by trasher** [comment]
Could you provide an example file (that can be used in glpi tests suite)?

**Comment by stonebuzz** [comment]
```imsi``` field never exist it's called ```subscriber_id```

**Comment by stonebuzz** [comment]
imsi and serial have been reversed

File: original.md [hunk 1]
```diff
@@ -755,10 +755,10 @@ Original XML specification
           -->
        <!ELEMENT TYPE (#PCDATA)>
 
-     <!ELEMENT SIMCARDS (IMSI, PHONE_NUMBER, ICCID, STATE, COUNTRY, OPERATOR_CODE,
+     <!ELEMENT SIMCARDS (SERIAL, PHONE_NUMBER, ICCID, STATE, COUNTRY, OPERATOR_CODE,
```
Comment (trasher) [inline original.md:758]: Original MD file should not be changed. Converter should handle rename from IMSI to serial.

File: tests/data/5.xml [hunk 2]
```diff
@@ -304,7 +304,7 @@ ARM Bios Ver 7.59u v46 454MHz B987-M995-F80-O0,0 MAC:00042d076b88</COMMENTS>
         </PORT>
       </PORTS>
       <SIMCARDS>
-        <ICCID />  <IMSI /></SIMCARDS>
+        <ICCID />  <SERIAL /></SIMCARDS>
```
Comment (trasher) [inline tests/data/5.xml:307]: I guess empty nodes can just be dropped

File: original.md (hunk 1 above)
Comment (stonebuzz) [reply original.md]: done !

File: tests/data/5.xml (hunk 2 above)
Comment (stonebuzz) [reply tests/data/5.xml]: done

**Review by trasher — APPROVED** [review]

**Pull request merged**